JWT/JWT.cs

```diff
@@ -60,31 +65,31 @@ public static string Encode(IDictionary<string, object> extraHeaders, object pay
             segments.Add(Base64UrlEncode(signature));
 
             return string.Join(".", segments.ToArray());
-        }
-
-        /// <summary>
-        /// Creates a JWT given a payload, the signing key, and the algorithm to use.
-        /// </summary>
-        /// <param name="payload">An arbitrary payload (must be serializable to JSON via <see cref="System.Web.Script.Serialization.JavaScriptSerializer"/>).</param>
-        /// <param name="key">The key used to sign the token.</param>
-        /// <param name="algorithm">The hash algorithm to use.</param>
+        }
+
+        /// <summary>
+        /// Creates a JWT given a payload, the signing key, and the algorithm to use.
+        /// </summary>
+        /// <param name="payload">An arbitrary payload (must be serializable to JSON via <see cref="System.Web.Script.Serialization.JavaScriptSerializer"/>).</param>
+        /// <param name="key">The key used to sign the token.</param>
+        /// <param name="algorithm">The hash algorithm to use.</param>
```

Comment:
not sure why this is different.  i made no changes.  i suspect it is a windows vs unix line feed thing.